bin/idam-user-token.sh

```diff
@@ -10,4 +10,4 @@
 ROLE="${1:-ccd-import}"
 USER_ID="${2:-1}"
 
-curl --silent http://localhost:4501/testing-support/lease -Fid="${USER_ID}" -Frole="${ROLE}"
+curl --silent http://localhost:5000/testing-support/lease -Fid="${USER_ID}" -Frole="${ROLE}"
```

Comment:
i don't think /lease is available in sidam. please double check.

Reply:
Right. Fixed with oauth flow.